components/brave_today/browser/brave_news_controller.cc

```diff
@@ -97,6 +98,13 @@ void BraveNewsController::ClearHistory() {
   // feed cache somewhere.
 }
 
+mojo::PendingRemote<mojom::BraveNewsController>
```

Comment:
Is this definitely needed? Desktop works without.

Reply:
i think it's needed. @SergeyZhukovsky wdyt?

Reply:
yes, we need to pass remote to Java https://github.com/brave/brave-core/pull/11121/files#diff-e3ba0a17d841d84c92148622a703b9dfc526ac4ca9c64dadb831716f18454cb9R23